<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_search">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>id</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>surname</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>isBudget</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>course</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_search">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск по</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_search">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_add">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_delete">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>id</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>surname</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>isBudget</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>course</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_delete">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить по</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ликвидировать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_delete">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить запись по id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_update_id">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_update_record">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getAllButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести все записи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadToXLSXButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Выгрузить поисковое окно в XLSX</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>База данных</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Создать или открыть существующую</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>ГОВНИЩЕ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
